### PR TITLE
Cleans up unused/dirty vars

### PR DESCRIPTION
**Pull request by ShizCalev**
Cleans up a ton of dirty edits on -alot- of airlocks/doors/windoors, air alarms, ect

Removes two unused door vars

Added documention for closeOtherId

Fixed AI control on execution room doors (var was set to 0 instead of 1 by accident)

### CODE REVIEW

File: code/game/machinery/doors/airlock_types.dm [hunk 1]
```diff
@@ -404,7 +404,7 @@
 	overlays_file = 'icons/obj/doors/airlocks/cult/runed/overlays.dmi'
 	assemblytype = /obj/structure/door_assembly/door_assembly_cult
 	hackProof = TRUE
-	aiControlDisabled = TRUE
+	aiControlDisabled = 1
```
Comment (ShizCalev) [inline code/game/machinery/doors/airlock_types.dm]: this isn't a boolean.

Comment (Qustinnus) [reply code/game/machinery/doors/airlock_types.dm]: 🤔 then that variable name is also kind of shitty

Comment (Qustinnus) [reply code/game/machinery/doors/airlock_types.dm]: also shouldnt we be using other defines/flags then instead of just numbers?

**Comment by Cyberboss** [comment]
@MapDiffBot

**Comment by Cyberboss** [comment]
@MapDiffBot

**Comment by Cyberboss** [comment]
@MapDiffBot

**Comment by Cyberboss** [comment]
@MapDiffBot

**Comment by Cyberboss** [comment]
@MapDiffBot

**Comment by Cyberboss** [comment]
@MapDiffBot

**Comment by Cyberboss** [comment]
@MapDiffBot

**Comment by Cyberboss** [comment]
@MapDiffBot

**Comment by Cyberboss** [comment]
@MapDiffBot

**Comment by Cyberboss** [comment]
@MapDiffBot

**Comment by Cyberboss** [comment]
@MapDiffBot

**Comment by Cyberboss** [comment]
@MapDiffBot

**Pull request merged**